pokemongo_bot/__init__.py

```diff
@@ -413,15 +413,13 @@ def current_inventory(self):
         items_stock = {x.value: 0 for x in list(Item)}
 
         for item in inventory_dict:
-            try:
-                # print(item['inventory_item_data']['item'])
-                item_id = item['inventory_item_data']['item']['item_id']
-                item_count = item['inventory_item_data']['item']['count']
+            item_dict = item.get('inventory_item_data', {}).get('item', {})
+            item_count = item_dict.get('count')
+            item_id = item_dict.get('item_id')
 
+            if not item_count and not item_id:
```

Comment:
I misunderstood the if condition. Previous one was `!= {}`, so the correct is `if item_count and item_id`, my bad.